rplugin/python3/chromatica/compile_args_database.py

```diff
@@ -190,7 +191,13 @@ def get_cdb_args(self, filename):
                     if _basename in arg:
                         continue;
                     else:
-                        res.append(arg)
+                        # if last added switch was standalone include then we need to append path to it
+                        if last == '-I' or last == '-isystem':
+                            res[len(res) - 1] += arg
```

Comment:
I think you need to apply
```
if not os.path.isabs(arg):
    arg = os.path.normpath(os.path.join(cwd, arg))
```
before appending it, as done in the cases where there are no spaces between `-I` or `-isystem`.

Reply:
I saw it, thank you for addressing it, but be aware that I am only a user and contributor and can’t approve your merge request.